acceptance_tests/lib/ChartMuseum.py

```diff
@@ -89,6 +89,25 @@ def upload_test_charts(self):
                     self.http_status_code_should_be(201, response.status_code)
             os.chdir('../')
 
+    def upload_test_more_version_charts(self):
```

Comment:
I see these new functions here, but are they being used in acceptance tests? (.robot files)

Reply:
not yet . I forget it .